### PR TITLE
Modify parser script to recreate target folder and run dita-ot command

### LINKED ISSUE #262: Copy ditamap to destination
In the `data` folder is an `index.ditamap` this is the `root` of the new document.

1. force delete any folder called `target` in the current folder
2. create a folder called `target` in the current folder
3. copy `index.ditamap` and `welcome.dita`  from `data` to `target/dita`
4. run the [command](https://github.com/DeepBlueCLtd/LegacyMan/issues/260) to export the ditamap to html.  Note: you'll have to download DITA-OT, and declare it's root folder as a DITA_OT environment variable  on your machine (note the switch from hyphen to underscore, hyphens don't work in env variables on mac/linux). I'm using version 3.7.4.
5. you will also need to have Java installed on your machine.
6. Insert a line in the console logging to report `Publish complete after xxx seconds. Root file at xxx\xxx\xxx`.  This will come after `parse complete` entry.

Note: for the output of this task, I'm expecting:
* an updated `parser.parser
* the `index,ditamap` and `welcome.dita`  in `data`
* a `target` folder that just contains the copied `index.ditamap` in the `target/dita` folder
* the published content in `target/html`.  No, it's not great to put generated content into version control, but it's a quick way for me to review/verify what has changed from the previous publish.

### PR DESCRIPTION
**Pull request by ebenezer-melkamu**
Fixes #262 

### Modifications
1. Modify the `parser.parser` script to remove `target` folder to recreate it and to run `dita-ot` command to transform the `index.ditamap` file `html`
2. Generated `html` file from the `index.ditamap` file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,16 +33,42 @@ python -m unittest discover
 
 Execute the following command from the project root directory to run program. The test_payload_for_parsed_json is optional parameter.
 
+### Install DITA-OT
+
+Before you can use the dita command, you need to download and install the `DITA-OT` package on your system. You can download the latest version of `DITA-OT` from the official [website](https://www.dita-ot.org/download). Once you have downloaded and extracted the package to a directory on your system, you can add the `DITA-OT/bin` directory to your `PATH` variable by adding the following line to your `.bash_profile file`:
```
Comment (IanMayo) [inline README.md]: second sentence of this paragraph is wrong - it should be explaining that we're about to check if dita is installed.

File: parser/parser.py [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
+    delete_directory(os.path.join(os.getcwd(), "target"))
+    create_directory("target")
+
+    #copy index.dita and welcome.dita from data dir to target/dita
+    source_dir = "data"
+    target_dir = os.path.join("target", "dita")
+    copy_files(source_dir, target_dir, ["index.ditamap", "welcome.dita"])
+
+    #Run DITA-OT command to transform the index.ditamap file to html
+    dita_command = ["dita", "-i", "./target/dita/index.ditamap", "-f", "html5", "-o", "./target/dita/html"]
```
Comment (IanMayo) [inline parser/parser.py]: no, the destination folder is `target/html`. please.

**Review by IanMayo — CHANGES_REQUESTED** [review]
Nice work, just a couple of small changes, please.

File: README.md (hunk 1 above)
Comment (ebenezer-melkamu) [reply README.md]: @IanMayo I have added an order for the instructions, i hope this makes it clear, please let me know if there is anything i have to fix

**Review by IanMayo — APPROVED** [review]

**Pull request merged**